### DISCUSSION TITLE
Desktop file does not show in menu because of capitilization.

### DISCUSSION THREAD

**Discussion by bkmo** [cstayyab/WALC#54] [Troubleshooting]
The .desktop file WALC.desktop does not show in the menu on KDE Plasma 5. If it is changed to walc.desktop (no capitilization) it is shown in the menu.  Probably not your bug, but an easy workaround.

**Comment by daffamumtaz2361** [comment]
This is very interesting problem indeed. If you configure KDE Menu Editor to show hidden entries, it's actually hidden. Usually I just move it to Network section and be done with it. If the desktop file is all lowercase and there is a Category entry on the desktop file then it would be displayed correctly. I'm going to investigate this a bit more
